repos/fdbt-site/src/pages/ticketRepresentation.tsx

```diff
@@ -37,6 +37,26 @@ const getFareTypeDesc = (fareType: TicketType) => {
     }
 };
 
+const getFareTypeHint = (fareType: TicketType) => {
+    switch (fareType) {
+        case 'period':
+            return [
+                'Unlimited travel within a geographical zone',
+                'Unlimited travel on specific service or set of services',
+            ];
+        case 'flatFare':
+            return [
+                'Single trip anywhere within a geographic zone',
+                'Single trip on a specific service or set of services',
+            ];
+        default:
+            return [
+                'Unlimited travel within a geographical zone',
+                'Unlimited travel on specific service or set of services',
+            ];
```

Comment:
We should look to remove the duplication of these options, that appear both in the `period` and `default` cases, we have two options:
* If these sentences are truly generic then we should just remove the `period` case as the component will just use the `default` case anyway. However the use of the `Unlimited travel` wording is the actual cause of this bug code change, so it being the default for all `fareTypes` might lead to this same issue re-occurring in the future.
* We change these `default` responses to be truly generic, this means that if a `fareType` that's not `period` or `flatFare` gets to this page (it's included in a new future features journey) then the wording seen is initially suitable, and could have it's own case created if needed.

Reply:
This is also done